.github/workflows/ci.yml

```diff
@@ -15,12 +15,31 @@ jobs:
         node-version: [14.x]
 
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout branch
+        uses: actions/checkout@v2
+      
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
-      - run: npx lerna bootstrap
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            node_modules
+            */*/node_modules
+            ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-
+      
+      - run: yarn install --frozen-lockfile
```

Comment:
let's keep it consistent with the following steps and run lerna bootstrap. with the change you've done in lerna.json it should already provide the frozen lockfile option. also, I'm not sure that this is doing what lerna bootstrap does and even though it may be doing it, it's not obvious if it is. lerna boostrap is doing more than what yarn install does, even when using workspaces. lerna is symlinking packages, and that's the way we should run it.

in other words, lerna boostrap implies yarn install, but not the other way around, or at least it's not obvious whether it is doing that or not.